cs-amazon/src/main/java/io/cloudslang/content/amazon/actions/network/CreateSubnetAction.java

```diff
@@ -0,0 +1,154 @@
+     *                         separated from query value by "="
+     *                         Examples: "parameterName1=parameterValue1&parameterName2=parameterValue2"
+     *                         Default: ""
+     * @param availabilityZone Optional - Availability Zone for the subnet.
+     *                         Default: AWS selects one for you. If you create more than one subnet in your VPC, we may
+     *                         not necessarily select a different zone for each subnet.
+     * @param vpcId            ID of the VPC
+     * @param cidrBlock        Network range for the subnet, in CIDR notation.
+     *                         Example: "10.0.0.0/24"
+     * @return A map with strings as keys and strings as values that contains: outcome of the action (or failure message
+     * and the exception if there is one), returnCode of the operation and the ID of the request
+     */
+    @Action(name = "Create Subnet",
+            outputs = {
+                    @Output(Outputs.RETURN_CODE),
+                    @Output(Outputs.RETURN_RESULT),
+                    @Output(Outputs.EXCEPTION)
+            },
+            responses = {
+                    @Response(text = Outputs.SUCCESS, field = Outputs.RETURN_CODE, value = Outputs.SUCCESS_RETURN_CODE,
+                            matchType = MatchType.COMPARE_EQUAL, responseType = ResponseType.RESOLVED),
+                    @Response(text = Outputs.FAILURE, field = Outputs.RETURN_CODE, value = Outputs.FAILURE_RETURN_CODE,
+                            matchType = MatchType.COMPARE_EQUAL, responseType = ResponseType.ERROR, isOnFail = true)
+            })
+    public Map<String, String> execute(@Param(value = ENDPOINT) String endpoint,
+                                       @Param(value = IDENTITY, required = true) String identity,
+                                       @Param(value = CREDENTIAL, required = true, encrypted = true) String credential,
+                                       @Param(value = PROXY_HOST) String proxyHost,
+                                       @Param(value = PROXY_PORT) String proxyPort,
+                                       @Param(value = PROXY_USERNAME) String proxyUsername,
+                                       @Param(value = PROXY_PASSWORD, encrypted = true) String proxyPassword,
+                                       @Param(value = HEADERS) String headers,
+                                       @Param(value = QUERY_PARAMS) String queryParams,
+                                       @Param(value = VERSION) String version,
+                                       @Param(value = AVAILABILITY_ZONE) String availabilityZone,
+                                       @Param(value = VPC_ID, required = true) String vpcId,
+                                       @Param(value = CIDR_BLOCK, required = true) String cidrBlock) {
+        try {
+            version = InputsUtil.getDefaultStringInput(version, "2016-09-15");
```

Comment:
this hardcoded value seems a bit fishy (I don;t know the logic behind it).. I think you should extract a const anyway